components/director/internal/tenantmapping/handler.go

```diff
@@ -52,25 +49,34 @@ type TenantRepository interface {
 	GetByExternalTenant(ctx context.Context, externalTenant string) (*model.BusinessTenantMapping, error)
 }
 
+// ClientInstrumenter describes instrumentation tools that support client instrumentation.
```

Comment:
better description would be:
`collects metrics for different client and auth flows